### PR TITLE
#159951816 Run codeclimate test coverage

### PR DESCRIPTION
**Pull request by RutaleIvanPaul**
#### What does this PR do?
Adds test coverage to code climate 
#### Description of Task to be completed?
* Setup circleci
* Add test coverage reporting
#### How should this be manually tested?
Trigger build on circleci
#### What are the relevant pivotal tracker stories?
[#159951816](https://www.pivotaltracker.com/story/show/159951816)

### CODE REVIEW

**Review by abulojoshua1 — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]

**Comment by lytes20** [comment]
Hey, @RutaleIvanPaul could please speak to why the coverage is to low?

**Review by ssewilliam — APPROVED** [review]

File: src/containers/Login/Login.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Login from './Login';
+import {shallow} from 'enzyme';
+
+it('renders right text', () => {
+
+    const wrapper = shallow((<Login />))
+    // const text = elem.find('div').children().first().text()
```
Comment (AGMETEOR) [inline src/containers/Login/Login.test.js]: Why is the commented out code still here?

File: src/containers/Login/Login.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    // expect(text).toEqual('Login form here.')
```
Comment (AGMETEOR) [inline src/containers/Login/Login.test.js]: Why is the commented out code still here?

**Comment by geofrocker** [comment]
Have  a folder inside src named tests put all the tests in that folder

**Comment by geofrocker** [comment]
Remove the package-lock.json file

**Review by AGMETEOR — APPROVED** [review]

**Comment by abulojoshua1** [comment]
@RutaleIvanPaul, do we need the snapshot files (for example src/tests/__snapshots__/Home.test.js.snap) in the PR?

**Comment by RutaleIvanPaul** [comment]
@abulojoshua1 , whenever I remove them, the build fails.

**Review by lytes20 — APPROVED** [review]

**Pull request merged**